Modify installation on Windows

an-tao opened the pull request.

K-ballo commented on drogon_ctl/CopyDlls.cmake:
```diff
@@ -0,0 +1,7 @@
+message(STATUS "dll:" ${DLL_FILES})
+message(STATUS "bin:" ${INSTALL_BIN_DIR})
+file(COPY ${DLL_FILES} DESTINATION ${INSTALL_BIN_DIR})
+file(COPY ${CMAKE_CURRENT_BINARY_DIR}/drogon_ctl.exe DESTINATION ${INSTALL_BIN_DIR})
```
This path refers to a file that does not exist for a multi-configuration generator (like Visual Studio).

an-tao replied:
I've installed drogon_ctl with visual studio (2019) successfully.

![图片](https://user-images.githubusercontent.com/21002501/75441140-1460a300-5998-11ea-84a2-34d656de130a.png)

The script is run by:
```cmake
add_custom_command(TARGET drogon_ctl POST_BUILD 
                   COMMAND ${CMAKE_COMMAND}
                   -DINSTALL_BIN_DIR=${INSTALL_BIN_DIR}
                   -P ${CMAKE_CURRENT_SOURCE_DIR}/CopyDlls.cmake )
```

K-ballo replied:
You are using a single-configurator generator (Ninja)

an-tao replied:
@K-ballo You are right, I've fixed it. Thanks!

K-ballo commented:
The install step fails because the 'webapp' example expects to find `gzip` in the path. Consider using CMake's own `-tar` command line tool.

After disabling the examples from being built, the install step succeeds, Drogon is found, and an application using it links correctly.

an-tao commented:
@K-ballo I removed the gzip command, the gz file is for #144 so it's can't be generated by the tar command in cmake, I just added a pre-generated index.html.gz file.
Thank you very much!

K-ballo commented:
> the gz file is for #144 so it's can't be generated by the tar command in cmake

I am not sure what this means, why would it be possible to generate this by calling `gzip` but not by calling `cmake -E tar`?

In any case, it's not a problem anymore.

an-tao commented:
> > the gz file is for #144 so it's can't be generated by the tar command in cmake
> 
> I am not sure what this means, why would it be possible to generate this by calling `gzip` but not by calling `cmake -E tar`?
> 
> In any case, it's not a problem anymore.

gzip compress the index.html directly, but `cmake -E tar cz` compress the tar file (the archivement of index.html). You could run `gzip -c index.html > index.html.gz` and `cmake -E tar cz index.html ...`, you'll find the results of the two command are different.
